boxtree/traversal.py

```diff
@@ -1805,10 +1805,6 @@ def get_kernel_info(self, dimensions, particle_id_dtype, box_id_dtype,
                                 "same_level_non_well_sep_boxes_starts"),
                             VectorArg(box_id_dtype,
                                 "same_level_non_well_sep_boxes_lists"),
-                            VectorArg(coord_dtype, "box_target_bounding_box_min",
-                                with_offset=False),
-                            VectorArg(coord_dtype, "box_target_bounding_box_max",
-                                with_offset=False),
```

Comment:
My read is that removing these entirely was premature; they do seem to get used via `load_true_box_extent` (maybe add a comment so nobody falls for this in the future?). We'll need to include them in the argument list, but you can do that conditionally if `targets_have_extent` (needs adjustment here and at the call site.